### ISSUE TITLE
Is this Project Dead?

### ISSUE THREAD

**Issue by emjimadhu** [mimecorg/vuido#63]

**Comment by mimecorg** [comment]
Yes and no. Vuido depends on libui and recently there's not much going on with it, so it's hard for this project to move much further. Proton Native, which is the equivalent of Vuido based on React, is currently migrating from libui to Qt; I would have to do the same with Vuido and I will when I have some time, but currently I'm trying to finish a big release of another project I'm working on.

**Comment by mestaritonttu** [comment]
For what it's worth, there is stuff going on in this tree for some months now: https://github.com/andlabs/libui/tree/remodel

**Comment by gtamas** [comment]
guys what`s up? 
Any news on this?

**Comment by frnco** [comment]
> Yes and no. Vuido depends on libui and recently there's not much going on with it, so it's hard for this project to move much further. Proton Native, which is the equivalent of Vuido based on React, is currently migrating from libui to Qt; I would have to do the same with Vuido and I will when I have some time, but currently I'm trying to finish a big release of another project I'm working on.

I think it's actually pretty funny that people stall like this instead of just contributing to the other project... Well I don't like neither Qt nor React so I guess it's pure Gtk3 for now... Still, kinda sad to not even be able to contribute because such a chain reaction causes so many projects to stall (libui -> libui-nodejs -> vuido)...

**Comment by cody271** [comment]
> Yes and no. Vuido depends on libui and recently there's not much going on with it, so it's hard for this project to move much further.

@mimecorg If still interested, please feel free to open any issues you have for upstream [here](https://github.com/libui-ng/libui-ng/issues). Thank you.

**Comment by bitsnaps** [comment]
Looks like the project is really dead now!

**Comment by mestaritonttu** [comment]
There is now a next gen libui: https://github.com/libui-ng/libui-ng

**Comment by arafatamim** [comment]
[vue-nodegui](https://github.com/nodegui/vue-nodegui) is probably a good alternative to this project.

**Comment by ozio** [comment]
@arafatamim nah, last commit 3 years ago
